php.yaml

```diff
@@ -68,35 +64,12 @@ subpackages:
   - name: php-fpm
     description: PHP 8.2 FastCGI Process Manager (FPM)
     pipeline:
-      - runs: |
-          mkdir -p ${{targets.subpkgdir}}/usr/bin
-          mv ${{targets.destdir}}/sbin/php-fpm ${{targets.subpkgdir}}/usr/bin/
-          mkdir -p ${{targets.subpkgdir}}/etc/php-fpm.d 
-          mv ${{targets.destdir}}/etc/php-fpm.conf.default ${{targets.subpkgdir}}/etc/php-fpm.conf
-          sed -i 's/\/home\/build\/melange-out\/php//g' ${{targets.subpkgdir}}/etc/php-fpm.conf
-          mv ${{targets.destdir}}/etc/php-fpm.d/www.conf.default ${{targets.subpkgdir}}/etc/php-fpm.d/www.conf \
-          && { \
-          	echo '[global]'; \
-          	echo 'error_log = /proc/self/fd/2'; \
-            echo 'daemonize = no'; \
-          	echo; \
-          	echo '[www]'; \
-            echo 'listen = [::]:9000'; \
-          	echo 'access.log = /proc/self/fd/2'; \
-            echo; \
-          	echo 'clear_env = no'; \
-          	echo; \
-          	echo 'catch_workers_output = yes'; \
-            echo; \
-          	echo 'decorate_workers_output = no'; \
-          } | tee ${{targets.subpkgdir}}/etc/php-fpm.d/zz-apko.conf
-
+      - runs: "mkdir -p ${{targets.subpkgdir}}/usr/bin\nmv ${{targets.destdir}}/sbin/php-fpm ${{targets.subpkgdir}}/usr/bin/\nmkdir -p ${{targets.subpkgdir}}/etc/php-fpm.d \nmv ${{targets.destdir}}/etc/php-fpm.conf.default ${{targets.subpkgdir}}/etc/php-fpm.conf\nsed -i 's/\\/home\\/build\\/melange-out\\/php//g' ${{targets.subpkgdir}}/etc/php-fpm.conf\nmv ${{targets.destdir}}/etc/php-fpm.d/www.conf.default ${{targets.subpkgdir}}/etc/php-fpm.d/www.conf \\\n&& { \\\n\techo '[global]'; \\\n\techo 'error_log = /proc/self/fd/2'; \\\n  echo 'daemonize = no'; \\\n\techo; \\\n\techo '[www]'; \\\n  echo 'listen = [::]:9000'; \\\n\techo 'access.log = /proc/self/fd/2'; \\\n  echo; \\\n\techo 'clear_env = no'; \\\n\techo; \\\n\techo 'catch_workers_output = yes'; \\\n  echo; \\\n\techo 'decorate_workers_output = no'; \\\n} | tee ${{targets.subpkgdir}}/etc/php-fpm.d/zz-apko.conf\n"
```

Comment:
This usually means there's stray whitespace somewhere — I think we can fix this before merging as-is

cc: @rawlingsj

Reply:
I think I've fixed this